cpg_workflows/jobs/trim.py

```diff
@@ -85,14 +87,20 @@ def __init__(
             raise ValueError(f'Invalid adapter type: {adapter_type}')
         self.command = [
             'cutadapt',
-            '-o', str(output_fastq_pair.r1),
-            '-a', adapters.r1.sequence,
+            '-o',
+            str(output_fastq_pair.r1),
+            '-a',
+            adapters.r1.sequence,
```

Comment:
This may be annoying enough that we should consider an annotation to suppress this reformatting for this statement.

Reply:
It's maybe more awkward, but you can reformat this as:

```python
[
    *('-o', str(output_fastq_pair.r1)),
    *('-a', adapters.r1.sequence),
    ...
]

Reply:
It's a bit awkward but I guess IMHO it's better than splitting the options from their arguments…